### DISCUSSION TITLE
Pycharm Copilot plugin not giving completions, logs out on each completion request

### DISCUSSION THREAD

**Discussion by timothyjlaurent** [community/community#19848] [Copilot Conversations]
I was happy using copilot during the beta, after it was generally available, my beta access expired and was getting 403 errors when trying to log in in the copilot plugin.

I signed up for commercial access, and logged in, using the activation code at this point copilot looks good:
![image](https://user-images.githubusercontent.com/2000204/176952892-34ebbfc3-b857-4bd5-9749-1aea50f8220a.png)

Then if I try to refresh completions in the plugin sidebar by clicking 'refresh':
![image](https://user-images.githubusercontent.com/2000204/176952971-e941ca46-2b98-44c3-b4bc-3e7afd5cc6ef.png)
It says "Retrieving completions"
![image](https://user-images.githubusercontent.com/2000204/176953048-2702125c-809a-4afb-888c-73218157ccc8.png)

but then the plugin logs out:
![image](https://user-images.githubusercontent.com/2000204/176953132-a564c5b4-90e3-4df7-b0ad-3f925208e40b.png)

I've tried logging out and then logging in, restarting pycharm, but still can't get any autocompletion working since I decided to start paying.

Please help me figure out how to get this working.

**Comment by huabin** [comment]
Hello @timothyjlaurent , this kind of issue is usually checked by the following steps.

1. Go to [https://github.com/settings/billing](https://github.com/settings/billing), check if you have paid for GitHub copilot. For most people, it now requires payment.
2. Remove GitHub Copilot from Pycharm and reinstall.
3. Sign in with a GitHub account that has been paid for GitHub copilot.

BTW, could you please provide more information?

- OS and its version
- Pycharm version
- GitHub Copilot plugin version

Thank you.

**Reply by DBZclaroty** [reply]
I'm also experiencing this issue.
- macOS Monterey 12.5.1
- PyCharm Professional 2022.2.1
- GitHub Copilot 1.1.31.1908

I've tried uninstalling GitHub Copilot and I still get logged-out on every attempt to "refresh", like in the case described by @timothyjlaurent .

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by DBZclaroty** [reply]
> Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

@davecheney  In my PyCharm version, there is no menu entry called "Output" under the 'View' menu.

What "relevant parts" are you looking for?

![image](https://user-images.githubusercontent.com/99262339/190358023-ec57b887-4559-44f8-91d3-93235ab50d93.png)

**Reply by DBZclaroty** [reply]
@davecheney  In the PyCharm logs, I see the same error mentioned in this ticket:  https://github.com/community/community/discussions/32285

**Reply by davecheney** [reply]
@DBZclaroty thank you. I suggest you follow up on https://github.com/orgs/community/discussions/32285
